dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/WsAddressStreamRealTimeTestSuite.scala

```diff
@@ -0,0 +1,59 @@
+package com.wavesplatform.it.sync.api.ws
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.api.ws.connection.WsConnection
+import com.wavesplatform.dex.api.ws.entities.{WsBalances, WsMatchTransactionInfo, WsOrder}
+import com.wavesplatform.dex.domain.account.KeyPair
+import com.wavesplatform.dex.domain.account.KeyPair.toAddress
+import com.wavesplatform.dex.domain.asset.Asset.Waves
+import com.wavesplatform.dex.domain.order.OrderType.{BUY, SELL}
+import com.wavesplatform.dex.model.OrderStatus
+import com.wavesplatform.it.WsSuiteBase
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+import scala.util.Using
+
+class WsAddressStreamRealTimeTestSuite extends WsSuiteBase with TableDrivenPropertyChecks {
```

Comment:
I think this test should do sth like - comparing timestamps & ensuring that delay between messages is much less than ```ws-messages-interval``` (we can set it 1 hour for example in order to make testing easier). it can be skipped this for now and done later.

Reply:
made address-actor.ws-messages-interval = 1 hour